lib/elements/satin_column.py

```diff
@@ -147,6 +158,17 @@ def pull_compensation(self):
         # wider than we desire the column to end up.
         return self.get_float_param("pull_compensation_mm", 0)
 
+    @property
+    @param(
+        'pull_compensation_balance',
+        _('Pull compensation balance'),
+        tooltip=_('Percentage of pull compensation which applies to second rail (50% is centered).'),
+        unit='%',
+        type='float',
+        default=50)
+    def pull_compensation_balance(self):
+        return min(100, max(0, self.get_float_param("pull_compensation_balance", 50)))
```

Comment:
This causes problems if it is 0 or is set back to zero.

Reply:
Setting this to 0 just causes the pull compensation to apply to only 1 side.

Reply:
```
  File "/lib/elements/satin_column.py", line 845, in do_center_walk
    forward, back = self.plot_points_on_rails(
  File "/lib/elements/satin_column.py", line 816, in plot_points_on_rails
    add_pair(pos0, pos1)
  File "/lib/elements/satin_column.py", line 747, in add_pair
    pos0, pos1 = self.offset_points(pos0, pos1, offset_px, offset_prop, offset_balance)
  File "/lib/elements/satin_column.py", line 704, in offset_points
    pos2 = pos2 + (pos2 - pos1).unit() * offset2
  File "/lib/utils/geometry.py", line 185, in unit
    return self.mul(1.0 / self.length())
ZeroDivisionError: float division by zero
```

Reply:
Fixed. Looks like the crash was due to a bug in `offset_points` caused by accidental variable shadowing. Try it now.

Reply:
Hmm, having this center at 50 might be weird for users.  Maybe it should be -100 to 100, with 0 as the center, or -1 to 1?

Reply:
I chose to have the center at 50 to be consistent with how the existing balance parameters in svg work. I think using a different convention might also cause confusion.

@lexelby do you have any ideas on how to improve the tooltip?